python/cudf/cudf/core/frame.py

```diff
@@ -1622,6 +1622,13 @@ def _repeat(self, count):
         result._copy_type_metadata(self)
         return result
 
+    def _reverse(self):
+        result = self.__class__._from_table(libcudf.copying.reverse(self))
+        return result
+
+    def reverse(self):
```

Comment:
We probably don't want a public `reverse()` method, as Pandas doesn't expose one.

Reply:
Note that `Frame` methods are exposed to end-users,  as the public classes `Series` and `DataFrame` and `Index` inherit from it.

Reply:
Makes sense, I will remove the public method.